utils/misc.py

```diff
@@ -90,4 +93,13 @@ def format_time(seconds):
         i += 1
     if f == '':
         f = '0ms'
-    return f
\ No newline at end of file
+    return f
+
+def save_current_code(path: str):
+    print(f"Saving current code to {path}")
+    root = HydraConfig.get().runtime.cwd
+    with zipfile.ZipFile(os.path.join(path, "files.zip"), "w", zipfile.ZIP_DEFLATED) as z:
+        for file_path in pathlib.Path(root).glob('**/*.py'):
```

Comment:
```suggestion
	for r,dirs,files in os.walk(root):
		dirs[:] = [d for d in dirs if d non in []]
		files[:] = [f for f in files if f.split('.')[-1] non in ['pyc']]
		for filename in files:
			file_path = os.path.join(root, filename)
```
Maybe something like this? So we can add the dir names and file extensions we want to be ignored more easily.